sdk/resourcemanager/Azure.ResourceManager.Core/src/ResourceGroupOperations.cs

```diff
@@ -334,8 +344,8 @@ public virtual async Task<ArmOperation<ResourceGroup>> StartAddTagAsync(string k
 
             var myResource = model as TrackedResource<TIdentifier>;
             TContainer container = Activator.CreateInstance(typeof(TContainer), ClientOptions, myResource) as TContainer;
-
-            return container.CreateOrUpdate(name, model);
+            var createOrUpdateMethod = typeof(TContainer).GetMethod(CreateOrUpdateMethodName);
+            return createOrUpdateMethod.Invoke(container, new object[] { name, model }) as ArmResponse<TOperations>;
```

Comment:
We can no longer call CreateOrUpdate directly because it is removed from base class, so I used reflection (same goes to CreateOrUpdateAsync).
Is there a better way?

Reply:
This method will most likely be moving to the GenericResourceContainer class where things will be created by id so I wouldn't worry too much about the implementation here.

Reply:
This worries me a bit.  Not positive that we can use the generic resource api in all cases.  Would it be preferrable to have an interface for creation?